extensions/mssql/src/hdfs/webhdfs.ts

```diff
@@ -794,7 +795,7 @@ export class WebHDFS {
 			canResume = false;
 			stream.on('resume', () => {
 				if (!canResume) {
-					stream._readableState.flowing = false;
+					(stream as any)._readableState.flowing = false; // i guess we are unsafely accessing this
```

Comment:
@kevcunnane - Thoughts? We should be safer here and add a check for _readableState being defined. And if it isn't then  probably emit an error. I imagine never an actual issue but better to be safe

Reply:
this is a workaround we inherited from the public WebHDFS module...  I think this is fine and overall an improvement. Happy to have a comment mentioning it'd be nice not to have to reference this.